CONTRIBUTING.md

```diff
@@ -205,3 +205,12 @@ Run `git config user.email` to see your Git email, and verify it with [your GitH
 webpack is feature rich and documentation is a time sink. We
 greatly appreciate any time spent fixing typos or clarifying sections in the
 documentation.
+
+## Testing with Docker
+
+Make sure to have installed docker before you continue.
+
+```sh
+$ docker build -t webpack-cli .
```

Comment:
I'd still recommend to add the `+https`, so the docker testing works until the migration is complete, we never know for sure when it will happen, so better have a working docker testing until then, for all the users, instead of having people to add the `+https` on their own  and not commit the changes